src/test/java/redis/clients/jedis/commands/jedis/StreamsCommandsTest.java

```diff
@@ -442,6 +441,28 @@ public void xack() {
       jedis.xack("xack-stream", "xack-group", range.get(0).getValue().get(0).getID()));
   }
 
+  @Test
+  public void xpendingRange() {
+    Map<String, String> map = new HashMap<>();
+    map.put("foo", "bar");
+    StreamEntryID m1 = jedis.xadd("xpendeing-stream", (StreamEntryID) null, map);
```

Comment:
null can not convert to `StreamEntryID`

Reply:
Not in this case - there are two different signatures for `xadd`, so It has to be explicitly converted.